nodewatcher/modules/defaults/network_profile/models.py

```diff
@@ -36,5 +36,6 @@ class RegistryMeta:
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('wifi-uplink', _("Use a wireless interface for uplink")))
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('no-wifi-ap', _("Disable wireless AP")))
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('no-wifi-mesh', _("Disable wireless mesh")))
+registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('wifi-mesh-80211s', _("Use 802.11s for meshing")))
```

Comment:
It is possible to enable by default but for existing nodes we would need a migration. A bigger problem is that then you would be able to check both options.

The best solution would be to instead introduce an additional field in the network profile model called `mesh_mode` or something, which would instead be a single choice field. And then use that in defaults.